deployment/mesh-infra/storage/pvc/kustomization.yaml

```diff
@@ -2,9 +2,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
```

Comment:
Document PVC stuff. Mention there's PVCs in correspondence of each storage class we've used so far.

Reply:
done